Gems/Atom/RHI/Vulkan/Code/Source/RHI/Allocator.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+#include <AzCore/base.h>
+#include <AzCore/PlatformIncl.h>
+#include <vulkan/vulkan.h>
+
+namespace AZ
+{
+    namespace Vulkan
+    {
+        class VkSystemAllocator 

```

Comment:
Add a comment explaining the purpose of this class.

Reply:
done.